master/cmd/determined-master/init.go

```diff
@@ -129,4 +129,6 @@ func registerConfig() {
 		defaults.Telemetry.SegmentMasterKey, "the Segment write key for the master")
 	registerString(flags, name("telemetry", "segment-webui-key"),
 		defaults.Telemetry.SegmentWebUIKey, "the Segment write key for the WebUI")
+	registerBool(flags, name("telemetry", "otel_enabled"), defaults.Telemetry.OtelEnabled, "enable otel")
+	registerString(flags, name("telemetry", "otel_endpoint"), defaults.Telemetry.OtelExportedOtlpEndpoint, "set otel endpoint")
```

Comment:
looking at the code above, it uses dash (e.g. `segment-webui-key`), so probably

```suggestion
	registerBool(flags, name("telemetry", "otel-enabled"), defaults.Telemetry.OtelEnabled, "enable otel")
	registerString(flags, name("telemetry", "otel-endpoint"), defaults.Telemetry.OtelExportedOtlpEndpoint, "set otel endpoint")
```

(and update the docs accordingly)